apps/workflows/tests/test_models.py

```diff
@@ -54,9 +54,9 @@ def test_function_value(self):
         """
         test if a function without parameters can be used as condition can be checked
         """
-        check = Check("is_major_incident_temp")
+        check = Check("major_incident_state")
         # alias
-        check_alias = Check("is major incident")
+        check_alias = Check("major incident state")
```

Comment:
@costaconrado Can I completely remove `test_function_value()` as there is no `is_major_incident_temp`?

Reply:
its fine to delete this test. 
Now that is_major_incident_temp don't exists we really don't have a good function in Flaw model to test this, so its fine to remove this test.